tsconfig.json

```diff
@@ -2,9 +2,6 @@
   "compilerOptions": {
     "experimentalDecorators": true,
     "module": "commonjs",
-    "declaration": true,
```

Comment:
Why did tsconfig change?

Reply:
I mentioned it in the commit message

This is to avoid warnings from `ts_library`